### ISSUE TITLE
Failure to copy generated alt text on perchance.org/ai-image-generator-

### ISSUE THREAD

**Issue by rthappe** [easrng/copy-alt-text#2]
1. Generate images on [perchance.org/ai-image-generator-](https://perchance.org/ai-image-generator-)
2. Wait for the result
3. Right-click one of the generated images
4. Copy Alt Text
5. Be disappointed: didn't work. (The error message advises to report that here.)

Firefox 146.0.1 (64-bit); Mozilla Firefox Snap for Ubuntu canonical-002 - 1.0; Ubuntu 22.04.5 LTS
